src/main/java/net/atos/zac/zoeken/converter/AbstractZoekObjectConverter.kt

```diff
@@ -0,0 +1,10 @@
+package net.atos.zac.zoeken.converter
+
+import net.atos.zac.zoeken.model.ZoekObject
+import net.atos.zac.zoeken.model.index.ZoekObjectType
+
+abstract class AbstractZoekObjectConverter<ZOEKOBJECT : ZoekObject> {
+    abstract fun supports(objectType: ZoekObjectType): Boolean
```

Comment:
Maybe add a constructor that takes in the `ZoekObjectType` that is supported, and implement this method right here.

Reply:
I might look at this later if you don't mind. :-)